src/sentry/integrations/issues.py

```diff
@@ -51,3 +51,13 @@ def get_create_issue_config(self, group, **kwargs):
                 'type': 'textarea',
             }
         ]
+
+    def get_link_issue_config(self, group, **kwargs):
+        return [
+            {
+                'name': 'externalIssue',
```

Comment:
good question. in all of our api endpoints, we use camel case i guess because then the frontend doesn't need to use snake case? honestly not sure of the reason exactly, but that's what we do.